README.md

```diff
@@ -38,7 +38,7 @@ recommend using the [app skeleton](https://github.com/cakephp/app) as
 a starting point. For existing applications you can run the following:
 
 ``` bash
-$ composer require cakephp/cakephp:"~3.6"
+$ composer require cakephp/cakephp:"~3.7"
```

Comment:
It would be better to use `3.7.*` as we do in skeleton app or simply remove the version from the command.